store/export_test.go

```diff
@@ -20,13 +20,43 @@
 package store
 
 import (
-	"github.com/snapcore/snapd/testutil"
+	"io"
+	"net/http"
+	"net/url"
 
+	"golang.org/x/net/context"
 	"gopkg.in/retry.v1"
+
+	"github.com/snapcore/snapd/overlord/auth"
+	"github.com/snapcore/snapd/progress"
+	"github.com/snapcore/snapd/snap"
+	"github.com/snapcore/snapd/testutil"
 )
 
 var (
 	HardLinkCount = hardLinkCount
+	ApiURL        = apiURL
+	Download      = download
+
+	UseDeltas  = useDeltas
+	ApplyDelta = applyDelta
+
+	GetCurrentSnap    = currentSnap
```

Comment:
Is there a clash with another symbol (CurrentSnap)?

Reply:
Yes, its rather unfortunate. I can do a followup for this. The "currentSnap" is a function and CurrentSnap is a type IIRC.